db/migrate/20180711025613_add_email_address_to_publication.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailAddressToPublication < ActiveRecord::Migration[5.2]
+  def change
+    add_column :publications, :reprintemails, :string
```

Comment:
I added `authors` to the WebOfScienceSourceRecords model with the intent of storing the email addresses there. I don't think we need this migration